src/components/SpiceList.js

```diff
@@ -1,16 +1,14 @@
 import React from 'react'
-// TODO: import the SpiceItem component here
+import SpiceItem from './SpiceItem';
 
 // class component
 class SpiceList extends React.Component {
 
   renderSpices() {
-    // TODO: update this function to return an array of SpiceItem components
-    // If you get an error message about a 'key' prop, make sure to fix it before submitting your code!
+    return this.props.spices.map(spice => <SpiceItem key={spice.id} spice={spice}/>)
```

Comment:
Perfect! If you want to see another option for passing down the spice object, one alternative would be to create a prop for each key on the spice object like this:

```jsx
this.props.spices.map(spice => <SpiceItem key={spice.id} id={spice.id} title={spice.title} image={spice.image} description={spice.description} notes={spice.notes} />)
```

This way you don't end up with a nested props object in the child component. You can also use a shorthand syntax with the spread operator to pass down all keys from an object as props:

```jsx
this.props.spices.map(spice => <SpiceItem key={spice.id} {...spice} />)
```

I'd say to be careful with that spread operator because it gives you less control over what props you're passing down, but it can be useful in some cases.